extensions/blocks/podcast-player/templates/podcast-header.php

```diff
@@ -52,6 +52,10 @@ class="jetpack-podcast-player__track-description"
 		<?php echo esc_attr( $track['description'] ); ?>
 	</div>
 	<?php endif; ?>
+
+	<div class="jetpack-podcast-player__audio-player">
+		<div class="jetpack-podcast-player--audio-player-loading"></div>
```

Comment:
I tried to see if we could remove this `<div class="jetpack-podcast-player--audio-player-loading"></div>` and just use a `:after` pseudo-element, but it looks like that would require some extra work. I like that this `div` gets automatically replaced by the react content, so I think this is good 👍

Reply:
All the static markup gets replaced so I wasn't too worried about it.